pallets/time-release/src/lib.rs

```diff
@@ -113,6 +153,21 @@ pub mod module {
 
 		/// The block-number provider.
 		type BlockNumberProvider: BlockNumberProvider<BlockNumber = BlockNumberFor<Self>>;
+
+		/// The aggregate call type
+		type RuntimeCall: Parameter
+			+ Dispatchable<
+				RuntimeOrigin = <Self as frame_system::Config>::RuntimeOrigin,
+				PostInfo = PostDispatchInfo,
+			> + From<Call<Self>>
```

Comment:
nit: indentation is confusing here

Reply:
Yeah, I agree, the linter is doing that.

Reply:
well poop on that linter